module/data/actor/character.mjs

```diff
@@ -14,6 +15,10 @@ const {
   ArrayField, BooleanField, HTMLField, IntegerSortField, NumberField, SchemaField, SetField, StringField
 } = foundry.data.fields;
 
+/**
+ * @typedef {import("./fields/simple-trait.mjs").SimpleTraitData} SimpleTraitData
```

Comment:
At least with typedoc, we can use a more ergonomic invocation. Might depend on your IDE whether it works for intellisense or not.

```suggestion
 * @import { SimpleTraitData } from "./fields/simple-trait.mjs";
```

Reply:
Our linter doesn't like `@import`, but I can add an exception for that. I was following core's styling for these, but I see you switched things up in V13 :).

Reply:
I think it was a relatively recent addition to typedoc